config/channel-settings.json

```diff
@@ -523,6 +666,7 @@
         "schema": {},
         "referenceLocation": {
           "customerAddressBusinessReferences": [
+            "CustomerId",
```

Comment:
Changing the default businessReference on an existing action will be a breaking change. We should discuss if this is necessary and, if so, how we'll migrate existing channel profiles and data in gateway-storage

Reply:
removed CustomerId from the default business references for now.